src/Stoa.ts

```diff
@@ -98,18 +98,12 @@ class Stoa {
                     return;
                 }
 
-                if (rows.length)
+                if (rows.length == 1)
```

Comment:
That's a weird change. What happen if we were to get `rows.length == 2` ?

Reply:
This is the function of returning a single validator value.
If `res.length ==2` return 204 to HTTP response code.

Reply:
But we look it up by address. We should NEVER have `res.length > 1`.

Reply:
Actually scratch that. Currently, it is possible that a validator uses multiple UTXOs to enroll.

Reply:
Oh, that's right.
I think I overlooked it.
I'll upload a new revision PR.

Reply:
If so, does the validator of the function return an enrollment of one of the UTXOs?

Reply:
I would say all of them, potentially sorted.